Examples/Python/src/Base.cpp

```diff
@@ -114,10 +114,10 @@ class PythonLogger {
 
  private:
   std::string m_name;
-  std::unique_ptr<const Logger> m_logger;
+  std::unique_ptr<const Acts::Logger> m_logger;
```

Comment:
Consider `using namespace Acts` which should be safe in `.cpp` files. Purely for conciseness.